### PR TITLE
Ansible File

### LINKED ISSUE #139: Ansible script to deploy multi image setup as Swarm Service
Assume that you localhost has a swarm manager running.

### PR DESCRIPTION
**Pull request by Manas23601**
Fixes #139

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,15 @@ For example, if you want to add a new Text Translation AI Model from OPENAI in r
       }
 }
 ```
+## Run Ansible Script
+Create a `inventory.ini` file which has the ssh details of the target machine.  
+Create a Hashicorp Vault to store your credentials such as `Github Username` and `Github Personal Access Token(PAT)`. Along with that you need to store your environment variables for the docker images.  
+Lastly you need to add the `Vault Addr` and `Vault Token` to your environment.
```
Comment (singhalkarun) [inline README.md]: It should clearly mention the names of variable to be added in environment of machine to keep it clear for user.

**Comment by Manas23601** [comment]
I couldn't find any better way to convert the hashicorp keys to a env file. I used a custom jinja2 template

File: swarm.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        env_file: "{{ env_file }}"
+
+    - name: get config
+      get_url:
+        url: https://raw.githubusercontent.com/Samagra-Development/ai-tools/restructure/config.json
+        dest: "{{ destination_directory }}/config.json"
+
+    - name: get script
+      get_url:
+        url: https://raw.githubusercontent.com/Samagra-Development/ai-tools/restructure/generate_independent_docker.sh
```
Comment (singhalkarun) [inline swarm.yml]: Can we take repository url also as a variable? As someone might fork it and still expect the whole setup to work? @Manas23601

Comment (Manas23601) [reply swarm.yml]: How can we ensure the code structure? After forking,  people might change the branch structure

**Pull request merged**